### PR TITLE
Add hive.hide-delta-lake-tables property

### PR DESCRIPTION
**Pull request by Ordinant**

### CODE REVIEW

**Review by mosabua — APPROVED** [review]
Looks good to me. Please ensure this gets merged before the 344 release @findepi

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 1]
```diff
@@ -335,6 +335,10 @@ Property Name                                      Description
 ``hive.metastore-refresh-max-threads``  Maximum threads used to refresh cached metastore data.        100
 
 ``hive.metastore-timeout``              Timeout for Hive metastore requests.                         ``10s``
+
+``hive.hide-delta-lake-tables``         Controls whether to hide Delta Lake tables in table          false
+                                        listings. Currently applies only when using the AWS Glue
+                                        catalog as metadata source.
```
Comment (findepi) [inline presto-docs/src/main/sphinx/connector/hive.rst:341]: Given the limitation, should we list this in Glue section for now? @electrum ?

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 2]
```diff
@@ -335,6 +335,10 @@ Property Name                                      Description
 ``hive.metastore-refresh-max-threads``  Maximum threads used to refresh cached metastore data.        100
 
 ``hive.metastore-timeout``              Timeout for Hive metastore requests.                         ``10s``
+
+``hive.hide-delta-lake-tables``         Controls whether to hide Delta Lake tables in table          false
+                                        listings. Currently applies only when using the AWS Glue
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/hive.rst:340]: We should say "when using the AWS Glue metastore" since to better match the existing language

**Review by electrum — APPROVED** [review]

File: presto-docs/src/main/sphinx/connector/hive.rst (hunk 1 above)
Comment (findepi) [reply presto-docs/src/main/sphinx/connector/hive.rst:341]: thoughts?

File: presto-docs/src/main/sphinx/connector/hive.rst (hunk 2 above)
Comment (Ordinant) [reply presto-docs/src/main/sphinx/connector/hive.rst:340]: Fixed. Added to the same commit with --append.

**Comment by Ordinant** [comment]
@electrum, the requested edit is in place.

Last Friday, there was some urgency to get this doc change in place for the 344 release. Can we merge this in now? Thanks!

**Comment by mosabua** [comment]
@findepi @electrum can you please merge?

**Comment by electrum** [comment]
Merged, thanks!

**Comment by findepi** [comment]
Thanks @electrum for the merge. Did you look into https://github.com/prestosql/presto/pull/5498#discussion_r502680956 as well?

**Pull request closed without merging**